lib/workers/repository/onboarding/pr/index.spec.ts

```diff
@@ -15,7 +15,7 @@ import type { BranchConfig } from '../../../types';
 import { OnboardingState } from '../common';
 import { ensureOnboardingPr } from '.';
 
-jest.mock('../../../../util/git');
+jest.mock('../../../../util/git'); //TODO: check why still needed!
```

Comment:
still not found the cause?

Reply:
found, indirect use git.getFile. todo removed.